code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -173,15 +173,23 @@
 \***************************************/
 /datum/action/changeling/weapon/arm_blade
 	name = "Arm Blade"
-	desc = "We reform one of our arms into a deadly blade. Costs 20 chemicals."
-	helptext = "We may retract our armblade in the same manner as we form it. Cannot be used while in lesser form."
+	desc = "We reform one of our arms into a deadly blade that breaks after a number of hits, improvable by absorbing genomes. Costs 40 chemicals."
+	helptext = "We may retract our armblade in the same manner as we form it. Organic tissue is not perfect; the armblade will break after it is used too much. The more genomes we absorb, the stronger it is. Cannot be used while in lesser form."
 	button_icon_state = "armblade"
-	chemical_cost = 20
-	dna_cost = 2
+	chemical_cost = 40
+	dna_cost = 4
 	req_human = TRUE
 	weapon_type = /obj/item/melee/arm_blade
 	weapon_name_simple = "blade"
 
+/datum/action/changeling/weapon/arm_blade/sting_action(mob/living/carbon/user)
+	var/datum/antagonist/changeling/changeling = IS_CHANGELING(user) //So we can read the absorbed_count.
+	if(!changeling)
+		return
+
+	var/obj/item/melee/arm_blade/blade = ..(user)
+	blade.remaining_uses = round(changeling.absorbed_count * 3)
```

Comment:
```suggestion
	var/obj/item/melee/arm_blade/blade = ..(user)
	blade.remaining_uses = round(changeling.absorbed_count * 3)
	return TRUE
```
untested code detected. because you don't return true this has 0 chemical cost associated rn

Reply:
incorrect, i literally tested it, it billed me 40 credits every use

Reply:
that seems unlikely 

![image](https://github.com/tgstation/tgstation/assets/51863163/659de480-23ff-4792-ae82-52712a034f95)

Reply:
i dont know what to tell you, worked on my machine

Reply:
i have pulled the code and it does not work, armblade is free

![image](https://github.com/tgstation/tgstation/assets/51863163/fd91a03c-0d1b-455f-96a5-4e3249f95374)

Reply:
i am so confused